envs/example/ci-full/group_vars/all.yml

```diff
@@ -16,13 +29,46 @@ neutron:
   lbaas:
     enabled: True
 
+common:
```

Comment:
That's why I wanted a playbook to maintain all these envs, to keep then consistent.